YOUR_PACKAGE/__main__.py

```diff
@@ -0,0 +1,44 @@
+"""
+Example of how to import and use the brewblox service
+"""
+
+import logging
+from typing import Type
+
+from aiohttp import web
+
+from brewblox_service import events, service
+
+routes = web.RouteTableDef()
+LOGGER = logging.getLogger(__name__)
+
+
+@routes.get('/hello')
```

Comment:
maybe more descriptive function names would be better?

hello -> '/endpoint_name'
greet -> 'endpoint_implementation'

Reply:
Fixed